src/crypto/kdf/blake2b.rs

```diff
@@ -0,0 +1,138 @@
+//! `blake2b` is the current default key derivation scheme of `libsodium`.
+
+use ffi;
+
+/// Number of bytes in a `Key`.
+pub const KEYBYTES: usize = ffi::crypto_kdf_blake2b_KEYBYTES as usize;
+
+/// Number of bytes in a `Context`
+pub const CONTEXTBYTES: usize = ffi::crypto_kdf_blake2b_CONTEXTBYTES as usize;
+
+/// Number of bytes in a `SubKey`.
+pub const SUBKEYBYTES: usize = 32;
```

Comment:
Why 32?

Reply:
Had to make a choice here for a fixed length type. The `pwhash` module does something similar for `Salt`. 32 seemed a reasonable choice since its equal to the keysize for the secretbox and secretstream APIs.

Do you prefer a dynamic-size type?

Reply:
I am not sure how to use it. I don't not what is kdf/blake2b for.

Reply:
KDF stands for key derivation function. You can use it to derive multiple independent subkeys from a single master key. This is used for example to derive rx and tx keys from a single key exchange.